src/components/Project/Services/content.js

```diff
@@ -494,7 +494,7 @@ export default class ServicesContent extends React.Component {
   }
 
   handleFormChanged() {
-    this.setState({userHasUnsavedChanges: true})
+    this.setState({ userHasUnsavedChanges: true, saving: false })
```

Comment:
quick question...is the `saving` parameter used anywhere?

Reply:
`saving` marks the `SAVE` button is disabled. If it is `false`, the button is clickable.